src/csvcubed/utils/sparql_handler/sparqlmanager.py

```diff
@@ -222,7 +210,7 @@ def select_csvw_dsd_dataset_label_and_dsd_def_uri(
 
 
 def select_csvw_dsd_qube_components(
-    csvw_shape: Optional[CSVWShape],
+    cube_shape: Optional[CubeShape],
```

Comment:
I don't think the cube shape should really be optional at this point, it should require a value.